packages/omg-js-rootchain/src/txUtils.js

```diff
@@ -48,24 +46,15 @@ async function sendTx ({ web3, txDetails, privateKey, callbacks }) {
         .on('error', reject)
     })
   }
-  return web3.eth.sendSignedTransaction(signedTx.rawTransaction).catch(err => {
-    let transactionHash
-    try {
-      transactionHash = JSON.parse(err.message.replace('Transaction has been reverted by the EVM:', '')).transactionHash
-    } catch (parseError) {
-      throw (err)
-    }
-    ethErrorReason({ web3, hash: transactionHash }).then(() => {
-      throw (err)
-    })
-  })
+  return web3.eth.sendSignedTransaction(signedTx.rawTransaction)
 }
 
 async function setGas (web3, txDetails) {
-  if (!txDetails.gasPrice) {
+  let enhancedTxDetails = { ...txDetails }
```

Comment:
Oh, this is sooo much nicer than the old, fugly `Object.assign()` way of copying an object 🙌